tests/migration_test.go

```diff
@@ -33,6 +33,8 @@ import (
 	"sync"
 	"time"
 
+	"kubevirt.io/kubevirt/tests/libinfra"
```

Comment:
nit: You could locate it closer to the tests group.

Reply:
I can't follow. If you mean I should move the import, `make generate` will do `goimports` on this and move it back?